translator/trace/jaeger/traces_to_jaegerproto.go

```diff
@@ -131,6 +130,10 @@ func attributesToJaegerProtoTags(attrs pdata.AttributeMap) []model.KeyValue {
 
 	tags := make([]model.KeyValue, 0, attrs.Cap())
 	attrs.ForEach(func(key string, attr pdata.AttributeValue) {
+		if key == conventions.AttributeServiceName {
+			return
+		}
```

Comment:
Thank you for fixing it! 

The only concern is that now we scatter handling of conventions.AttributeServiceName across many functions, it doesn't seem right especially having `attributesToJaegerProtoTags` being used in other places. Probably add another argument to `attributesToJaegerProtoTags` like `skipKeys map[string]bool` that can be used to filter particular keys out. What do you think?

cc @tigrannajaryan

Reply:
`attributesToJaegerProtoTags` is used in several places, so it makes sense.

Reply:
I do not like this. Map lookup is going to have cost. Translation functions need to be fast, and this one loop is the most frequently called one. I would like to see performance profile before accepting this.

Reply:
I think map lookups should be pretty fast. map initialization is the most expensive, but here it's handled perfectly, only two initializations. Otherwise we can just duplicate the function and create `resourceAttributesToJaegerProtoTags` for now in order to not block the fix. @tigrannajaryan what do you think?

Reply:
If we want a quick fix let's pass a single `bool` parameter that controls `AttributeServiceName` specifically. 

If we want the map I can be easily convinced by a CPU profile that shows it is not significant. :-)